keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/config.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Pascal Jaeger <pascal.jaeger@leimstift.de>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define CRC8_USE_TABLE
+#define CRC8_OPTIMIZE_SPEED
+
+/* Reset to bootloader */
+#define RESET_PIN                                  GP22
+#define RESET_PIN_DOUBLE_TAP_RESET_TIMEOUT         1000U
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED     GP25
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+/* split settings */
+#define SPLIT_POINTING_ENABLE
+#define SPLIT_HAND_PIN GP28 // high = left, low = right
+
+/* serial configuration for split keyboard */
+#define SERIAL_PIO_USE_PIO0
```

Comment:
This isn't needed, nor even used.  if `SERIAL_PIO_USE_PIO1` isn't defined, it's assumed to be pio0. 
```suggestion
```